rosdep/python.yaml

```
@@ -6867,6 +6867,15 @@ python3-mock:
       packages: [mock]
   rhel: ['python%{python3_pkgversion}-mock']
   ubuntu: [python3-mock]
+python3-moonraker-api-pip:
+  debian:
+    '*':
+      pip:
+        packages: [moonraker-api]
+  ubuntu:
+    '*':
+      pip:
+        packages: [moonraker-api]
```

```suggestion
python3-moonraker-api-pip:
  debian:
    pip:
      packages: [moonraker-api]
  ubuntu:
    pip:
      packages: [moonraker-api]
```

When a definition applies to all distributions the distribution field can be elided.